internal/pubsub/gcppubsub/gcppubsub.go

```diff
@@ -80,51 +145,28 @@ func (s *subscription) ReceiveBatch(ctx context.Context) ([]*driver.Message, err
 		}
 		ms = append(ms, m)
 	}
-	return ms, err
+	return ms, nil
 }
 
-// SendAcks acknowledges the messages with the given ackIDs on the server so
-// that they will not be received again for this subscription if the server
-// gets the acks before their deadlines. This method returns only after all
-// the ackIDs are sent, an error occurs, or the context is cancelled.
+// SendAcks implements driver.Subscription.SendAcks.
 func (s *subscription) SendAcks(ctx context.Context, ids []driver.AckID) error {
 	var ids2 []string
 	for _, id := range ids {
-		id2, ok := id.(string)
-		if !ok {
-			return fmt.Errorf("gcppubsub driver bug: cast from driver.AckID to string failed on %v", id)
-		}
+		id2 := id.(string)
```

Comment:
Now you can just inline the type assertion into the following line.

Reply:
Done.